### PR TITLE
Add xstream@7.0.0 w/ npm autoupdate

### PR DESCRIPTION
**Pull request by ian4hu**
# Profile of the lib
 * Git repository (required): https://github.com/staltz/xstream/
 * Official website (optional, not the repository): http://staltz.com/xstream/
 * NPM package url (optional): https://www.npmjs.org/package/xstream
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 739
   - Count of watchers: 39
   - Count of forks: 40
 * NPM download stats (optional):
   - Downloads in the last day: 185
   - Downloads in the last week: 1,083
   - Downloads in the last month: 5,165

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/xstream/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "license": "MIT",
+  "keywords": [
+    "xstream",
+    "reactive stream",
+    "functional"
+  ],
+  "dependencies": {
+    "symbol-observable": "^1.0.2"
+  },
+  "config": {
```
Comment (PeterDaveHello) [inline ajax/libs/xstream/package.json]: Please remove "config", thanks.

File: ajax/libs/xstream/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "xstream",
+  "version": "7.0.0",
+  "description": "An extremely intuitive, small, and fast functional reactive stream library for JavaScript",
+  "typings": "index.d.ts",
```
Comment (PeterDaveHello) [inline ajax/libs/xstream/package.json]: Please remove "typeings"

File: ajax/libs/xstream/package.json (hunk 1 above)
Comment (ian4hu) [inline ajax/libs/xstream/package.json]: I use npm test, the result is passed....

I will remove it soon.

Comment (PeterDaveHello) [inline ajax/libs/xstream/package.json]: Sorry that we can not recognize all the useless fields right now, may need to use something like whitelist in future, sorry for the inconvenience caused.

**Comment by kennynaoh** [comment]
@ian4hu 
Would you like to add `http://staltz.com/xstream/` as `homepage` filed in package.json?
Thanks!

**Comment by ian4hu** [comment]
@kennynaoh the homepage link is added

**Comment by PeterDaveHello** [comment]
@ian4hu thanks

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**